### PR TITLE
Gwh apidocs 1606 ima i os android

### PR DESCRIPTION
**Pull request by genwhittTTD**
IMA plugins for iOS and Android

### CODE REVIEW

File: docs/guides/mobile-plugin-ima-android.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+**GWH_SW do you know the repo for this?**
+
+## Requirements 
+
+To run this plugin, install the following:
+
+1. Google IMA SDK v3.29.0 ([ONLY 3.29.0 not any higher version](https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side/history))
+1. [UID2 Android SDK v0.4.0](../sdks/uid2-sdk-ref-android.md)
+1. UID2 IMA Plugin for Android v0.1.0
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-android.md:32]: Done, thx!

File: docs/guides/mobile-plugin-ima-android.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Version
+
+<!-- As of 2023-07-15 -->
+
+This documentation is for the UID2 IMA Plugin for Android version 0.1.0.
+
+## GitHub Repository
+
+**GWH_SW do you know the repo for this?**
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-android.md:24]: Done, thx!

File: docs/guides/mobile-plugin-ima-android.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# UID2 IMA Plugin for Android Integration Guide
+
+The  UID2 Interactive Media Ads (IMA) Plugin for Android enables publishers that use the [Google Interactive Media Ads SDK](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/) to include UID2 data in ad requests from Android apps. This is done automatically, with no direct coding needed from the app developer.
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-android.md]: Yep thx. Included SS comment too, hope it's correct, if not LMK.

File: docs/guides/mobile-plugin-ima-ios.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+## Version
+
+<!-- As of 2023-07-15 -->
+
+This documentation is for the UID2 IMA Plugin for iOS version 0.1.0.
+
+
+## GitHub Repository
+
+**GWH_SW do you know the repo for this?**
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-ios.md:25]: Done, thx!

File: docs/guides/mobile-plugin-ima-ios.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+This plugin is in the following open-source GitHub repository:
+
+- [https://github.com/IABTechLab/uid2-ios-plugin-google-ima](https://github.com/IABTechLab/uid2-ios-plugin-google-ima)
+
+## Requirements 
+
+To run this plugin, install the following:
+
+1. Google IMA SDK v3.18.5 **GWH_SW hoping to get a link for this**
```
Comment (sunnywu) [inline docs/guides/mobile-plugin-ima-ios.md:32]: https://developers.google.com/interactive-media-ads/docs/sdks/ios/client-side/history

**Comment by sunnywu** [comment]
Looks good, after you add the ios ima sdk link you can just merge it without more changes. (no other page should link to these IMA pages so should be ok to publish?)

File: docs/guides/mobile-plugin-ima-ios.md (hunk 5 above)
Comment (genwhittTTD) [reply docs/guides/mobile-plugin-ima-ios.md:32]: Done. Thx!

**Comment by genwhittTTD** [comment]
Publishing the IMA files renamed to be excluded from build -- pending a technical fix so we can make them visible.

**Pull request merged**